pkg/sriov/sriov.go

```diff
@@ -91,7 +92,12 @@ func (s *sriovManager) SetupVF(conf *sriovtypes.NetConf, podifName string, cid s
 		// Save the original effective MAC address before overriding it
 		conf.OrigVfState.EffectiveMAC = linkObj.Attrs().HardwareAddr.String()
 
-		if err = s.nLink.LinkSetHardwareAddr(linkObj, hwaddr); err != nil {
+		// Retry up to 5 times; wait 200 milliseconds between retries
```

Comment:
can you add a nice comment here explaining why we add this wait function here.

Reply:
Done.